.env

```diff
@@ -0,0 +1,5 @@
+TELEGRAM_API_TOKEN=5183312668:AAHIg2uT60CxzMMlC4yIOm4fKlTPMTIWzvQ
+DATABASE_URL=mysql://127.0.0.1:3306/tedexbot
```

Comment:
This should be `PLANETSCALE_PRISMA_DATABASE_URL` and also commit this as `.env.sample`.

Don't commit your actual `.env` in the future.